src/main/java/taxi/controller/LogoutController.java

```diff
@@ -0,0 +1,18 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = "/logout")
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
```suggestion
    public void doGet(HttpServletRequest req, HttpServletResponse resp)
```
better to make these overridden methods public